src/server/pfs/fuse/fuse_test.go

```diff
@@ -579,7 +579,7 @@ func TestMountDir(t *testing.T) {
 		require.Equal(t, "bar", filepath.Base(files[0].Name()))
 		require.Equal(t, "foo", filepath.Base(files[1].Name()))
 
-		//require.ElementsEqualUnderFn(t, []string{"foo", "bar"}, []string{"foo", "bar"}, func(f interface{}) interface{} { return f.(iofs.FileInfo).Name() })
+		// require.ElementsEqualUnderFn(t, []string{"foo", "bar"}, []string{"foo", "bar"}, func(f interface{}) interface{} { return f.(iofs.FileInfo).Name() })
```

Comment:
Is this an auto formatting change from the IDE?

Reply:
Yeah it is, you are right